### DISCUSSION TITLE
Is it possible to self-host lowcoder for free? Is it possible to create native Android and IOS applications?

### DISCUSSION THREAD

**Discussion by rogeriojpp** [lowcoder-org/lowcoder#575] [Ideas]
Is it possible to create native applications for Android and iOS using lowcoder? If so, how is the application published, the lowcoder documentation is very confusing.

Is it possible to self-host lowcoder for free?

**Comment by FalkWolsky** [comment]
Yes, it is possible without any restrictions or limits to self-host Lowcoder.
You can find here more Information: https://docs.lowcoder.cloud/lowcoder-documentation/setup-and-run/self-hosting

Mobile Applications are separated in 4 areas:

1) Web Application with Responsive Character (Can be displayed on Desktop and Mobile)
2) Web Application with Mobile Dimensions (makes sense only to display on Mobile)
3) Web Application with Mobile-optimized Components (coming in Lowcoder v2.5)
4) True native iOS and Android Apps - not yet supported. We think we start this year to make it possible - but it will cost us at least 6 months to realize this)

**Comment by FalkWolsky** [comment]
We need to update:

3) Web Application with Mobile-optimized Components (coming in Lowcoder v3.x)
4) True native iOS and Android Apps - are supported now. We have a service provider which we will activate in an upcoming release - and with it's help you can turn a Lowcoder App into a mobile native app - incl publishing on the Apple Store and Google Play

**Reply by ScottKemp** [reply]
Exciting! Thank you

**Comment by m8jj** [comment]
Is it possible yet to create true native iOS and Android apps? Would love to know how to do this as we are looking to be able to at least create a PWA if we can't do native with Lowcoder.
